src/transformers/tokenization_pegasus.py

```diff
@@ -47,8 +47,8 @@ class PegasusTokenizer(ReformerTokenizer):
     pretrained_vocab_files_map = PRETRAINED_VOCAB_FILES_MAP
     max_model_input_sizes = PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, *args, pad_token="<pad>", **kwargs):
```

Comment:
I really don't understand why Pegasus Tokenizer inherits from Reformer. I don't think they have anything in common...but well

Reply:
Indeed, let’s cut the link at some point.